modules/wordads/wordads.php

```diff
@@ -23,28 +23,30 @@ class WordAds {
 	 * @var array
 	 */
 	public static $ad_tag_ids = array(
-		'mrec'           => array(
+		'mrec' => array(
 			'tag'    => '300x250_mediumrectangle',
 			'height' => '250',
 			'width'  => '300',
 		),
-		'lrec'           => array(
```

Comment:
What's the reasoning behind getting rid of that tag ID? It is not in use anymore?

Reply:
I believe it is no longer supported.

Description:

> * […] removed `lrec` (no longer supported)

See also: pafL3P-5e-p2 #comment-324